src/main/scala/com/ing/wbaa/rokku/sts/api/AdminApi.scala

```diff
@@ -62,4 +64,26 @@ trait AdminApi extends LazyLogging with Encryption {
     }
   }
 
+  def enableOrDisableAccount: Route =
+    put {
+      path("account" / Segment / ("enable" | "disable")) { account =>
+        authorizeToken(verifyAuthenticationToken) { keycloakUserInfo =>
+          extractUri { uri =>
+            if (userInAdminGroups(keycloakUserInfo.userGroups)) {
+              val action = uri.path.toString.split("/").last match {
```

Comment:
you have the status in the `account` variable

Reply:
changed to uid which is more meaningful